### DISCUSSION TITLE
[Bug] Scoop fails to update with git

### DISCUSSION THREAD

**Discussion by DevelopCMD** [ScoopInstaller/Scoop#5670] [Help and Questions] [answered]
#### Current Behavior
Running "scoop update -a" results in an error: git: 'remote-https' is not a git command. See 'git --help'.

#### Expected Behavior
Scoop updates all the repos as normal.

#### Additional context/output
Output:
C:\Users\User>scoop update -a
Updating Scoop...
git: 'remote-https' is not a git command. See 'git --help'.
Update failed.

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** 5.1 revision 1776

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "last_update":  "2023-07-01T11:30:16.2564360-05:00",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by DevelopCMD** [comment]
it's been 3 hours and no one has replied to this issue. this is a really bad issue, please help

**Comment by HUMORCE** [comment]
Any additional configuration of git?

`git config --list --global`

**Comment by DevelopCMD** [comment]
running the command results in
"fatal: unable to read config file '/home/User/.gitconfig': No such file or directory"

**Comment by DevelopCMD** [comment]
because of this, i won't be able to update/install apps using scoop for a while

**Comment by HUMORCE** [comment]
> "fatal: unable to read config file '/home/User/.gitconfig': No such file or directory"

Are you using Scoop in WSL or Git Bash? please provide a full reproduction steps.

And
```
which git
get-command git
```

**Comment by DevelopCMD** [comment]
i have it installed using powershell

**Comment by DevelopCMD** [comment]
i have git installed with scoop

**Comment by rashil2000** [comment, marked as answer]
Seems like you're using a really old version of Git, that was installed along with the one from Scoop - and which has preference over Scoop's git. You need to uninstall that older git.

**Reply by HBDeveloper11** [reply]
The version of git that comes with Scoop is really old, so I should uninstall the old git and reinstall it using "scoop install git".

**Reply by DevelopCMD** [reply]
Note: after reinstalling git, the issue still persists. please help

**Reply by rashil2000** [reply]
What's the output of `git --version`?

**Comment by DevelopCMD** [comment]
Apparently, l found the issue.
For some reason, git-remote-https was blacklisted by my antivirus, so all I had to do was restore it and add an exception.
Everything works perfectly fine now.

**Reply by HBDeveloper11** [reply]
Note: the antivirus I use is Avast, and for some reason it thought git-remote-https was a virus, even though it's not. Be careful when you use Avast, because it might detect false positives!

**Reply by DevelopCMD** [reply]
and this is the reason why i had to uninstall avast
